starcluster/commands/start.py

```diff
@@ -264,8 +264,8 @@ def execute(self, args):
                     "Cannot use --dns-prefix when the cluster tag contains "
                     "a dot.")
             scluster.dns_prefix = tag
-        if self.opts.dns_sufix:
-            scluster.dns_sufix = tag
+        if self.opts.dns_suffix:
+            scluster.dns_suffix = self.opts.dns_suffix
```

Comment:
As a matter of fact, I just realize that I could have used the functionality as is but renaming my clusters to match the "zone" I need inside the network. This works as aliases are built as {}.{} when suffix is specified.

I can change back to the boolean implementation, which would work just fine. Would keep the typo fix and support for the DNS_SUFFIX option in the configuration (although it's not as important).

I think this would be preferable.